src/OnlineRetailPortal.Core/Translators/Translator.cs

```diff
@@ -0,0 +1,67 @@
+﻿using OnlineRetailPortal.Contracts;
+using OnlineRetailPortal.Core;
+using System;
+using System.Linq;
+
+namespace OnlineRetailPortal.Core
+{
+    public static class Translator
+    {
+        public static AddProductStoreRequest ToEntityRequest(this Product product)
+        {
+            AddProductStoreRequest  entityStoreRequest = new AddProductStoreRequest() {
+                Id = product.Id,
+                Name = product.Name,
+                Description = product.Description,
+                HeroImage = new Contracts.Image { Url = product.HeroImage.Url},
+                Price = new Contracts.Price { Amount = product.Price.Amount, IsNegotiable = product.Price.IsNegotiable},
+                Category = (Contracts.Category)product.Category.GetHashCode(),
```

Comment:
why gethashcode()??

Reply:
Was trying something. Forgot to remove that. Removed.
Done